code/game/objects/items.dm

```diff
@@ -634,12 +641,30 @@
 		//START MONKEY
 		var/mob/living/carbon/monkey/MO = M
 		switch(slot)
+			if(slot_head)
+				if(MO.hat)
+					return 0
+				if( !(slot_flags & SLOT_HEAD) )
+					return 0
+				return 1
 			if(slot_wear_mask)
 				if(MO.wear_mask)
 					return 0
 				if( !(slot_flags & SLOT_MASK) )
 					return 0
 				return 1
+			if(slot_glasses)
+				if(MO.glasses)
+					return 0
+				if( !(slot_flags & SLOT_EYES) )
+					return 0
+				return 1
+			if(slot_w_uniform)
+				if(MO.uniform)
+					return 0
+				if( !(slot_flags & SLOT_ICLOTHING) )
```

Comment:
They're in with the regular operators; no distinction is made
Which is stupid, but oh well